site/contents/write_ups/2022/06-04-22_writeup.md

```diff
@@ -0,0 +1,76 @@
+
+### How does White Saviourism show up in the pursuit of tech solutions for social good?
+
+One of our members did a dissertation which involved reading Google's AI ethics blog posts. One thing that came up (even before ideas of responsible AI) was a trend of "AI for Good" which in some ways is White Saviourism inate. The general approach was "if the country isn't in the west, let's go there and solve the problem with the AI." 
+
+One of the problems is that there is a desire to shoehorn AI (or more generally, tech) into solutions. There are lots of problems where we need more money, people to help etc. and instead people in the tech sector are like "AI will fix it" (e.g. predictive policing, hiring and firing algorithms or [worker chat apps designed to suppress discussions of worker's rights](https://theintercept.com/2022/04/04/amazon-union-living-wage-restrooms-chat-app/)).
+
+This made some of us think of events like Red Nose Day where often people (usually in the West) don't want to solve problems, they just see it as a PR opportunity. It is the only time the poorest people in the world are represented. 
+
+One academic specific problem is when money is ring fenced for international help/charity in general. Don't really fully understand how companies would go about doing the same thing or why would they want to unless to distract data? Is it basically just a tax write off for them? If they were trying to be good people then would it not be hard to optimise by not focussing on using technology.
+
+### What Western ideals influence the way we build technology in the Global North? What should we be doing to learn more effectively from other parts of the world?
+
+We are all so busy trying to solve problems but we are not asking if they are the right problems to solve. We want money and we want to look philantropic. We also want to look cool solving problems (shoutout to [Elon Musk and his submarine](https://www.bbc.co.uk/news/world-us-canada-50667553) comes to mind). 
+
+Under what circumstances should we do businesses in Africa/abroad in general? Providing goods and services? There's models of partnerships with governments/community groups that are the purely extractive neo-colonial models which are too prevelant. There has to be a fine line there because if there's a corrupt government/group you're working with, how do we know the money we're putting in is going to the right place? An example a little closer to home is UK PPE contracts where there was [‘apparent systemic bias’ in award of lucrative PPE deals favouring firms connected to Tories](https://www.theguardian.com/world/2021/apr/22/fifth-of-uk-covid-contracts-raised-red-flags-for-possible-corruption).
```

Comment:
```suggestion
Under what circumstances should we do businesses in Africa/abroad in general? Providing goods and services? There's models of partnerships with governments/community groups that are the purely extractive neo-colonial models which are too prevelant. There has to be a fine line there because if there's a corrupt government/group you're working with, how do we know the money we're putting in is going to the right place? An example a little closer to home is UK PPE contracts where there was [‘apparent systemic bias’ in award of lucrative PPE deals favouring firms connected to Tories](https://www.theguardian.com/world/2021/apr/22/fifth-of-uk-covid-contracts-raised-red-flags-for-possible-corruption), so this issue is one that is prevalent everywhere.
```